### DISCUSSION TITLE
Excel Data Import

### DISCUSSION THREAD

**Discussion by jdani015** [dbeaver/dbeaver#21094] [Q&A] [answered]
### Description

Trying to import data from Excel into a table.  Only file format option available is CSV.  

### DBeaver Version

CE 23.1.5

### Operating System

Windows 10

### Database and driver

Snowflake JDBC driver

### Steps to reproduce

1. From Database Navigator right-click on target table name
2. select "Import Data"
3. Data Transfer pop-up window not showing Excel file format as an option.  Only .CSV is displayed

### Additional context

DBeaver Office Support is installed.

**Comment by E1izabeth** [comment, marked as answer]
Importing from xlsx files is a feature available in PRO version only.
You can try trial version https://dbeaver.com/download/

**Reply by jdani015** [reply]
Thank you for clarifying.  On the download page I see choices Lite, EE and Ultimate.  Which of these is considered PRO?

**Reply by E1izabeth** [reply]
All of them

**Comment by jdani015** [comment]
👍
